UI/obs-app.cpp

```diff
@@ -768,7 +768,12 @@ bool OBSApp::MigrateGlobalSettings()
 		return false;
 	}
 
-	std::filesystem::copy(legacyGlobalConfigFile, userConfigFile);
+	try {
+		std::filesystem::copy(legacyGlobalConfigFile, userConfigFile);
+	} catch (const std::filesystem::filesystem_error &error) {
+		OBSErrorBox(nullptr, "Unable to migrate global configuration - copy failed.\n%s", error.what());
```

Comment:
I'm sorry that I didn't spot this sooner, but the error box is not a good place to hold the exception description as it's a customer-facing dialog.

It would neither be readable nor be of any value to the end user.

A developer could set a breakpoint there and inspect the `error` object, so no information is lost.

So the simple "Unable to.." message suffices.